### ISSUE TITLE
How to load external data libs

### ISSUE THREAD

**Issue by anandrikka** [brian-dawn/gitbook-plugin-klipse#4]
I have tried to load using the following snippet directly in .md file, but it is not working. Could you please help me out.

"
```eval-js
<pre class="hidden"><code class="lang-eval-clojure">
(def a 42)
</code></pre>
<pre class="hidden"><code class="lang-eval-js" data-external-libs="https://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-dom.min.js">
ReactDOM.render(
<h1>Hello, World !!</h1>,
document.getElementById('root')
);
</code></pre>
```

"

**Comment by brian-dawn** [comment]
Hi! I unfortunately do not know much about evaling JS with klipse. I attempted to mess with it locally, some things I noticed:

I am unsure if you want the `pre` class to be hidden.
I noticed you're using JSX, I think you might need to change your language to jsx but I am not sure.
You might have more luck asking on the klipse page https://github.com/viebel/klipse

I managed to find http://blog.klipse.tech/javascript/2016/12/14/jsx.html examining the source might be helpful but I didn't have any luck getting it to work right away...

Sorry I can't be of more help!
